src/middlewares/checkData.js

```diff
@@ -0,0 +1,58 @@
+import Response from '../helpers/response';
```

Comment:
it will be better if you rename your middlewares names that are associated with task or feature, because checkData is too common can be used on every routes

Reply:
Yeah, it is one validation file that includes many...(I and Gustave will use it to group login and sign up middlewares) and I think its better. It is an object which will contain multiple functions to verify different things. and of course, the naming of function is based on what they do